WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -23709,7 +23709,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.automattic.jetpack;
 				PRODUCT_MODULE_NAME = WordPress;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "WPiOS Development Profile";
+				PROVISIONING_PROFILE_SPECIFIER = "Jetpack iOS Development";
```

Comment:
@Gio2018 is this an intended change? Because it seems to be affecting the WordPress `pbxproj`.

Reply:
Yes it is, it only affects the Jetpack target

Reply:
To better clarify this answer, the change here sets the Jetpack iOS Development profile on the Jetpack target, needed to build on a device.